README.md

```diff
@@ -89,6 +89,15 @@ To enable retry upon test failure please set `TEST_RETRIES` environment variable
 TEST_RETRIES=2 yarn test
 ```
 
+### Docmosis Tornado
```

Comment:
I am not sure it is the right place. With `H3` it makes it part of `Testing` section which IMO is not related to running tests. I mean not directly.

Reply:
On second thought whole readme is bit wired because in getting started it focuses on E2E tests.

Reply:
Yeah to be honest I was not sure where to put it, would use some cleaning.

Reply:
Did second level header above testing.,